src/Ryujinx.Horizon/Arp/Ipc/Reader.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Ryujinx.Horizon.Common;
+using Ryujinx.Horizon.Sdk.Arp;
+using Ryujinx.Horizon.Sdk.Arp.Detail;
+using Ryujinx.Horizon.Sdk.Ns;
+using Ryujinx.Horizon.Sdk.Sf;
+using Ryujinx.Horizon.Sdk.Sf.Hipc;
+using System;
+
+namespace Ryujinx.Horizon.Arp.Ipc
+{
+    partial class Reader : IReader, IServiceObject
+    {
+        private readonly ApplicationInstanceManager _applicationInstanceManager;
+
+        public Reader(ApplicationInstanceManager applicationInstanceManager)
+        {
+            _applicationInstanceManager = applicationInstanceManager;
+        }
+
+        [CmifCommand(0)]
+        public Result GetApplicationLaunchProperty(out ApplicationLaunchProperty applicationLaunchProperty, ulong applicationInstanceId)
+        {
+            // NOTE: Returns InvalidArgument if applicationLaunchProperty is null, doesn't occur in our case.
+            //       Returns InvalidPointer if applicationInstanceId is null, doesn't occur in our case.
+
+            if (_applicationInstanceManager.Entries[applicationInstanceId] == null)
+            {
+                applicationLaunchProperty = new ApplicationLaunchProperty();
```

Comment:
```suggestion
                applicationLaunchProperty = default;
```
For consistency with what is done on other places.